src/m365/spo/commands/tenant/tenant-commandset-list.ts

```diff
@@ -0,0 +1,43 @@
+import { Logger } from '../../../../cli/Logger';
+import { CommandError } from '../../../../Command';
+import { formatting } from '../../../../utils/formatting';
+import { odata } from '../../../../utils/odata';
+import { spo } from '../../../../utils/spo';
+import { urlUtil } from '../../../../utils/urlUtil';
+import SpoCommand from '../../../base/SpoCommand';
+import commands from '../../commands';
+import { ListItemInstanceCollection } from '../listitem/ListItemInstanceCollection';
+
+class SpoTenantCommandSetListCommand extends SpoCommand {
+  public get name(): string {
+    return commands.TENANT_COMMANDSET_LIST;
+  }
+
+  public get description(): string {
+    return 'Retrieves a list of ListView Command Sets that are installed tenant-wide';
+  }
+
+  public defaultProperties(): string[] | undefined {
+    return ['Title', 'TenantWideExtensionComponentId', 'TenantWideExtensionWebTemplate'];
```

Comment:
I think it makes more sense if we show the `TenantWideExtensionListTemplate` property instead of the WebTemplate.